apps/hash-graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -336,12 +413,18 @@ impl<'p> Column<'p> {
             Self::PropertyTypes(_) => Table::PropertyTypes,
             Self::EntityTypes(_) => Table::EntityTypes,
             Self::Entities(_) => Table::Entities,
-            Self::PropertyTypeDataTypeReferences(_) => Table::PropertyTypeDataTypeReferences,
+            Self::PropertyTypeDataTypeReferences(_) => {
+                Table::ReferenceTable(ReferenceTable::PropertyTypeDataTypeReferences)
+            }
             Self::PropertyTypePropertyTypeReferences(_) => {
-                Table::PropertyTypePropertyTypeReferences
+                Table::ReferenceTable(ReferenceTable::PropertyTypePropertyTypeReferences)
+            }
+            Self::EntityTypePropertyTypeReferences(_) => {
+                Table::ReferenceTable(ReferenceTable::EntityTypePropertyTypeReferences)
+            }
+            Self::EntityTypeEntityTypeReferences(_) => {
+                Table::ReferenceTable(ReferenceTable::EntityTypeInheritance)
             }
```

Comment:
I may be a little out of the loop, but we expect to add an explicit reference table for `EntityTypeLinks` as well here, right?

Reply:
Yes, this is done in #2202. Currently, the reference table is the same.